pkg/util/asynclog/async_log.go

```diff
@@ -0,0 +1,116 @@
+/*
+Copyright 2022 The Koordinator Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package asynclog
+
+import (
+	"bytes"
+	"io"
+	"os"
+	"sync"
+	"sync/atomic"
+
+	"github.com/spf13/pflag"
+	"k8s.io/klog/v2"
+)
+
+var (
+	enableAsync = pflag.Bool("async-log", false, "enable to async write log and must enable logtostderr")
```

Comment:
When these two flags will be added to the program's binary file, and when the application should consider enabling asynchronous logging, a comment is needed to explain the usage precautions.

Reply:
update.